plugins/plotly-express/src/PlotlyExpressChartPanel.tsx

```diff
@@ -20,27 +20,72 @@ export interface PlotlyExpressChartPanelProps extends ChartPanelProps {
 function PlotlyExpressChartPanel(props: PlotlyExpressChartPanelProps) {
   const dh = useApi();
   const { fetch, ...rest } = props;
+  const containerRef = useRef<HTMLDivElement>(null);
+  const [model, setModel] = useState<PlotlyExpressChartModel>();
 
   const makeModel = useCallback(async () => {
     const widgetInfo = await fetch();
     const data = getWidgetData(widgetInfo);
     const { plotly, deephaven } = data;
     const isDefaultTemplate = !deephaven.is_user_set_template;
     const tableColumnReplacementMap = await getDataMappings(widgetInfo);
-    return new PlotlyExpressChartModel(
+    const m = new PlotlyExpressChartModel(
       dh,
       tableColumnReplacementMap,
       plotly.data,
       plotly.layout ?? {},
       isDefaultTemplate,
       ChartTheme
     );
+    setModel(m);
+    return m;
   }, [dh, fetch]);
 
+  useEffect(
+    function handle3DTicks() {
+      if (!model || !containerRef.current || !model.has3D()) {
+        return;
+      }
+
+      const container = containerRef.current;
+
+      function handleMouseDown() {
+        model?.pauseUpdates();
+      }
+
+      function handleMouseUp() {
+        model?.resumeUpdates();
+      }
+
+      let wheelTimeout = 0;
+
+      function handleWheel() {
+        model?.pauseUpdates();
+        window.clearTimeout(wheelTimeout);
+        wheelTimeout = window.setTimeout(() => {
+          model?.resumeUpdates();
+        }, 300);
+      }
+
+      container.addEventListener('mousedown', handleMouseDown);
+      container.addEventListener('mouseup', handleMouseUp);
```

Comment:
Do you still get a `mouseup` if you click and drag outside of the container and release the mouse?

Reply:
Good catch. I added a window `mouseup` listener when the `mousedown` happens which is triggered even if the mouse leaves the browser window entirely